cypress/support/e2e.ts

```diff
@@ -0,0 +1,20 @@
+// ***********************************************************
```

Comment:
this file has no reason to be in the root

Reply:
It looks like this file is necessary. Without it I get the following error
![image](https://github.com/Idrinth/api-bench/assets/77408854/23cf179e-2e67-42c1-b8f1-6d1918efdf0f)

```
Your project does not contain a default supportFile. We expect a file matching cypress/support/e2e.{js,jsx,ts,tsx} to exist.

If a support file is not necessary for your project, set supportFile to false.

https://on.cypress.io/support-file-missing-or-invalid

[Stack trace](http://localhost:57920/__launchpad/index.html#)
Error: Your project does not contain a default supportFile. We expect a file matching cypress/support/e2e.{js,jsx,ts,tsx} to exist.

If a support file is not necessary for your project, set supportFile to false.

https://on.cypress.io/support-file-missing-or-invalid
    at L (:4165:9336)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async D.buildBaseFullConfig (:4388:64661)
    at async D.getFullInitialConfig (:4388:66110)
    at async D.setupNodeEvents (:4388:60237)
```

Reply:
I'm not sure what you are doing there, likely trying to start cypress in the wrong folder. Cypress files in the root make no sense

Reply:
I see what happened. I removed this.